client/reader/following-edit/index.jsx

```diff
@@ -299,6 +309,16 @@ var FollowingEdit = React.createClass( {
 		);
 	},
 
+	toggleAddSite: function() {
+		this.setState( {
+			isAddingOpen: ! this.state.isAddingOpen
+		} );
+
+		if ( ! this.state.isAddingOpen ) {
+			document.getElementById( 'search-component-1' ).focus();
```

Comment:
fixed by delaying the render of the search form until we need it and turning autoFocus back on